dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/association/jdbc/JdbcOrgUnitAssociationsStore.java

```diff
@@ -58,6 +58,11 @@ public SetValuedMap<String, String> getOrganisationUnitsAssociationsForCurrentUs
     {
         Set<String> userOrgUnitPaths = getUserOrgUnitPaths();
 
+        if ( objectUids.isEmpty() )
+        {
+            return new HashSetValuedHashMap<>();
+        }
+
         return jdbcTemplate.query(
```

Comment:
What about the other method `getOrganisationUnitsAssociations`, should we have the same fix there, as it seems it has the same weakness?